use-cases/website-authorization/README.md

```diff
@@ -0,0 +1,159 @@
+## OAuth Website Authorization
+
+The authorization model for the Selling Partner API is based on Login with Amazon: Amazon's implementation of OAuth 2.0. In this model your application is authorized through interactions with pages displayed by Amazon and by your website.
+
+The website authorization workflow (OAuth) is initiated from your own website. Selling partners sign in to your website and select an Authorize button that you configure to initiate authorization. For more information, refer to the following steps
+
+We recommend the following resources for more details on authorization with the Selling Partner API:
+[Authorizing Selling Partner API Applications](https://developer-docs.amazon.com/sp-api/docs/authorizing-selling-partner-api-applications)
+[Website Authorization Workflow](https://developer-docs.amazon.com/sp-api/docs/website-authorization-workflow)
+
+## Solution
+
+This sample solution is an implementation of the Login with Amazon website authorization workflow. This web app can be deployed as a sample developer website and configured with your developer application to initiate the authorization workflow, from the selling partner clicking on an Authorize button until retrieving the Refresh Token and executing a first successful API call.
+
+The solution works in the following steps:
+1 - A landing page with an Authorize button is visited
+2 - The selling partner clicks on the Authorize button. The web app constructs the OAuth URL and redirects the selling partner
+3 - The selling partner consents and authorizes your website on seller/vendor central. Amazon redirects the user to the configured redirect URI (website) with the selling partner authorization code
+4 - The web app exchanges the authorization code for a refresh token
+5 - The web app exchanges the refresh token for an access token and calls the Sellers API operation to get the selling partner marketplace participations.
+
+Note:
+- The solution implements authorization for an application in the DRAFT status. For this, a **version=beta** query parameter is added to the OAuth URL. If your application is already in status PUBLISHED, simply remove this parameter.
```

Comment:
Can you please add a link to the resource and line that has to be updated for published apps?